http/handler_acp.go

```diff
@@ -47,11 +41,7 @@ func (s *acpHandler) AddPolicy(rw http.ResponseWriter, req *http.Request) {
 }
 
 func (s *acpHandler) AddDocActorRelationship(rw http.ResponseWriter, req *http.Request) {
-	db, ok := req.Context().Value(dbContextKey).(client.DB)
-	if !ok {
-		responseJSON(rw, http.StatusBadRequest, errorResponse{NewErrFailedToGetContext("db")})
-		return
-	}
+	db := mustGetContextClientDB(req)
```

Comment:
question: Why is this turned into a `must` instead of a `try` like what you did is some other handles?

Reply:
Only one that remained a `try` (not panic) is the P2P one because I think in that case it makes sense because P2P might be turned off and I see that specific error being returned by original author of the p2p code as useful for that situation as useful.

Rest all context casts are converted to a `must`.